recipes/zmqpp/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans import AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+class ZmqppConan(ConanFile):
+    name = "zmqpp"
+    homepage = "https://github.com/zeromq/zmqpp"
+    version = "4.2.0"
+    version_major = 4
```

Comment:
This can be obtained from `tools.Version(self.version)` currently this prevents the recipe from working with v5